packages/common/src/interfaces/gridOption.interface.ts

```diff
@@ -461,6 +461,12 @@ export interface GridOption {
    */
   ignoreAccentOnStringFilter?: boolean;
 
+  /**
+   * Defaults to false, should we ignore any accent while sorting text?
+   * For example if our text is "José" and we type "Jose" then it won't return unless we use this flag because "é" is not equal to "e"
+   */
+  ignoreAccentOnStringSort?: boolean;
```

Comment:
hmm I think we should probably rename the previous flag and keep only 1, maybe `ignoreAccentOnStringFilterAndSort`